### ISSUE TITLE
Error needs cleaning up for input file not existing

### ISSUE THREAD

**Issue by mr-c** [ged-lab/khmer#718]
```
|| This is the script 'extract-paired-reads.py' in khmer.
|| You are running khmer version 1.3-rc3
|| You are also using screed version 0.7
||
|| If you use this script in a publication, please cite EACH of the following:
||
||   * MR Crusoe et al., 2014. http://dx.doi.org/10.6084/m9.figshare.979190
||
|| Please see http://khmer.readthedocs.org/en/latest/citations.html for details.

Traceback (most recent call last):
  File "/usr/local/bin/extract-paired-reads.py", line 5, in <module>
    pkg_resources.run_script('khmer==1.3-rc3', 'extract-paired-reads.py')
  File "build/bdist.linux-x86_64/egg/pkg_resources.py", line 528, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "build/bdist.linux-x86_64/egg/pkg_resources.py", line 1394, in run_script
    execfile(script_filename, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/khmer-1.3_rc3-py2.7-linux-x86_64.egg/EGG-INFO/scripts/extract-paired-reads.py", line 154, in <module>
    main()
  File "/usr/local/lib/python2.7/dist-packages/khmer-1.3_rc3-py2.7-linux-x86_64.egg/EGG-INFO/scripts/extract-paired-reads.py", line 86, in main
    check_file_status(args.infile, args.force)
  File "/usr/local/lib/python2.7/dist-packages/khmer-1.3_rc3-py2.7-linux-x86_64.egg/khmer/file.py", line 25, in check_file_status
    mode = os.stat(file_path).st_mode
OSError: [Errno 2] No such file or directory: '*.pe.*.abundfilt'
```

**Comment by mr-c** [comment]
@jessicamizzi How is this going?

**Comment by mr-c** [comment]
Fixed by #772
